### PR TITLE
Update ESLint dependency

### PR DESCRIPTION
**Pull request by abdonrd**
#### Description of what you did:

Update ESLint dependency.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [ ] Documentation
- [x] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ module.exports = {
   rules: {
     'generator-star-spacing': 0,
     'no-console': 0,
+    'require-atomic-updates': 0,
```
Comment (abdonrd) [inline .eslintrc.js]: If anyone wants to eliminate this rule, go ahead!

**Comment by soupette** [comment]
@abdonrd your PR doesn't passes the tests... Can you fix them?

**Comment by abdonrd** [comment]
@soupette ops! There is an issue with `babel-eslint`: https://github.com/babel/babel-eslint/issues/791

Pinned the ESLint v6.1.0 version should work now!

**Comment by abdonrd** [comment]
@soupette green no! 😄

**Comment by soupette** [comment]
It looks good for me but will need @alexandrebodin insight also since there are a lot of backend files that have been modified

**Comment by abdonrd** [comment]
@alexandrebodin done!

**Comment by alexandrebodin** [comment]
@abdonrd Thank you ! we will wait for the merging of th develop branch and update this branch to merge it !

**Comment by alexandrebodin** [comment]
@abdonrd If you feel like updating this one ;)

**Comment by abdonrd** [comment]
@alexandrebodin updated!

**Comment by alexandrebodin** [comment]
@abdonrd a short yarn install should fix the last conflicts :)

**Comment by abdonrd** [comment]
@alexandrebodin done!

**Comment by alexandrebodin** [comment]
@abdonrd  Going to wait for @soupette approval then merge ;)

**Pull request merged**